src/assets/scripts/client/math/core.js

```diff
@@ -236,4 +236,18 @@ export const generateRandomOctalWithLength = (length = 1) => {
     }
 
     return leftPad(value.join(''), length);
+};
+
+/**
+ * Generates a random number, on an (approximate) bell curve.
+ * AKA - Normal Distribution, Gaussian Distribution
+ * 
+ * @function getNormalDistributedNumber
+ * @param average
+ * @return {number}
+ */
+export const getNormalDistributedNumber = (average) => {
+    const initialNumber = ((Math.random() + Math.random() + Math.random() + Math.random() + Math.random() + Math.random()) % 6);
```

Comment:
why the repeated `Math.random()`s and what is the significance of the `% 6` at the end?  this would be a good spot for an inline comment for those less fluent in math (like myself)

Reply:
interesting way to do that. This would produce a proper gaussian random distribution (aka the classic "bell curve"). And then the assumption is that you would measure the difference from `3` at the end, and use that to shift the wind up and down. The resulting behavior would be that it would usually have very slight increases or decreases, with dramatic changes possible, but far less likely.

The trouble with trying to do just `Math.random()` is that this function produces an even distribution, meaning that `0, 0.1, 0.5, 0.77, 0.89, 1` are all just as likely to be selected as each other. When trying to emulate wind changes, this would just be too jittery.

This does seem heavy for what it does, but it actually seems to be one of the best performing ways to get it done. We could reduce it from 6 to like 3 or 4, losing some integrity, but I'm not sure the time spent on this is really much of a concern anyway. Would probably be best to keep it accurate, and just call it every couple of seconds or something-- definitely not every frame.

Reply:
If you're interested: 
https://jsperf.com/normal-random-number-generators/8